Servlets/src/main/java/GetNameServlet.java

```diff
@@ -0,0 +1,21 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class GetNameServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        PrintWriter writer = response.getWriter();
+        // получаем параметр id
```

Comment:
(cosmetic) No comments in languages other than English, please) . Learn to "think in English". You will need it anyway